.github/actions/build-test/entrypoint.sh

```diff
@@ -2,14 +2,15 @@
 
 cd /github/workspace/source && mkdir build && cd build || exit 1
```

Comment:
Does this exit silently unlike the other ones? I am not actually sure it can fail so I wonder the need of `|| exit 1`

Reply:
Hm yeah I think I was following a shellcheck warning where apparently it's good practice to abort if a process fails after `cd ...`. I guess it's to prevent the script from continuing to run in the wrong directory. It's a bit overkill here because it would indeed exit fatally anyway.

Reply:
This is the relevant warning: https://github.com/koalaman/shellcheck/wiki/SC2164
I'm keeping it for now, it does no harm.